arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,642 @@
+          {
+            "name": "registryName",
+            "in": "path",
+            "description": "The container registry name.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RegistryCredentials"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "RegistryNameCheckRequest": {
+      "description": "The request to check whether the container registry name is available.",
+      "required": [
+        "name",
+        "type"
+      ],
+      "properties": {
+        "name": {
+          "description": "The container registry name.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The container registry resource type.",
+          "type": "string"
+        }
+      }
+    },
+    "RegistryNameStatus": {
+      "description": "The result of checking name availability.",
+      "properties": {
+        "nameAvailable": {
+          "description": "The boolean value that indicates whether the name is available.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "reason": {
+          "description": "The reason that the container registry name could not be used.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "The error message that explains the reason value in more detail.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Registry": {
+      "description": "The container registry.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RegistryProperties",
+          "description": "The container registry properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RegistryProperties": {
+      "description": "The container registry properties.",
+      "properties": {
+        "loginServer": {
+          "description": "The URL to log into the container registry.",
+          "type": "string",
+          "readOnly": true
+        },
+        "creationDate": {
+          "format": "date-time",
+          "description": "The creation date of the container registry in ISO8601 format.",
+          "type": "string",
+          "readOnly": true
+        },
+        "adminUserEnabled": {
+          "description": "The boolean value that indicates whether admin user is enabled.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "storageAccount": {
+          "$ref": "#/definitions/StorageAccountBase",
+          "description": "The storage account base properties.",
+          "readOnly": true
+        }
+      }
+    },
+    "StorageAccountBase": {

```

Comment:
We used StorageAccount for a different purpose in our RP and here StorageAccountBase, as a response to the user, will only contain storage account name.